### ISSUE TITLE
[1.11.2] Startup crash (w/ Nether Metals error as well)

### ISSUE THREAD

**Issue by TheCorinthians** [MinecraftModDevelopment/EndMetals#30]
I honestly could not figure out what the problems are, I'm pretty sure I have all the mods updated to their latest versions as well as Forge...
https://pastebin.com/ehpLK4i1

Relevant Versions Used:
BaseMetals - 2.5.0-beta4.113
EndMetals - 1.2.0-beta1.10
NetherMetals - 1.2.0-beta1.21
OreSpawn - 3.2.2.135
Forge 2588

I must be missing something completely obvious but, I just can't really to see it myself...

**Comment by TheCorinthians** [comment]
I looked back at my mistake from the other startup crash that was easily fixed with adding its missing mod but this time, I have everything that's needed (or at least that I believe is needed)... still not sure what's going on here...

**Comment by dshadowwolf** [comment]
... Okay - we have some more backport work to do. Late January to mid-February we changed the creative tabs of *Metals to be fully dynamic and it looks like that change - made to the 1.12 code base - has been backported to 1.11.2, but only for BaseMetals and ModernMetals, apparently.

@jriwanek you're the one in charge of backports - think we can save our userbase some headaches here ?

**Comment by TheCorinthians** [comment]
Yeah, and again, apologies for the fact that it's a slightly older version... slightly...  :)

**Comment by dshadowwolf** [comment]
Nah - we're actually trying to support as much as possible for the current series of releases, just because we like our userbase and think "just upgrade to the new MineCraft" is not a proper answer.

**Comment by TheCorinthians** [comment]
Lol, I really appreciate that. Though, the reason why I haven't gotten to 1.12 yet is because there's several great mods that have made it to 1.11.2 but not 1.12.2 such as Mystcraft, Torcherino, Waila and Dense Ores (unless Dense Metals can support more mod compatibility/integrations, probably coming soon, I presume?)...

**Comment by dshadowwolf** [comment]
Waila itself is, kinda, under new management - see HWYLA (a fork/continuation)

RWTema hasn't ported DenseOres - I think the new registration system of 1.12 has made it nearly impossible to make work the way it did. DenseMetals... needs a lot of work - @knoxhack has done a great job with it, but it needs work to be all it could be.

Torcherino... never played with it and Mystcraft, well... It was fun in 1.7 but I got hooked on RFTools Dimensions and never looked back 😃

**Comment by TheCorinthians** [comment]
Ah, interesting...
Well, Torcherino speeds up tile entities (furnaces smelt faster, plants/crops grow faster, etc.), a very convenient mod

_Questions:_
So, HWYLA works just like Waila? What are the differences?

RFT Dimensions, does it work similarly to Mystcraft? I may look into it...

_Suggestion!_
You know what you guys should add to Dense Metals that Dense Ores used to do? Mainly in its 1.8.9 versions, there was Overworld Ore Deposits! This is what they were... 

> Added Ore Overworld Deposits. Large structures of ore that contain large amounts of ore that generate very rarely on the surface.

You should totally add this feature to Dense Metals! Unless, RWTema won't allow that...

**Comment by dshadowwolf** [comment]
HWYLA is WAILA - the code was forked to keep it maintained and ported and has seen a lot of cleanup and improvement

**Comment by TheCorinthians** [comment]
Ah, ok, I'll start using that after I update later... though, you should really consider the suggestion for Dense Metals... Can I put that as a pull request in their page? Or is that not what a pull request is? I honestly don't even know how that works, I'm still new to Github, lol

**Comment by dshadowwolf** [comment]
I'll talk with Knox - he's part of the MMD Team, even if he's on a break to handle RL stuff. Its likely that the team will step in for DenseMetals at some point.

**Comment by TheCorinthians** [comment]
Nice, I'm really looking forward to it!

**Comment by TheCorinthians** [comment]
Hey... it's been over a month now... have you guys been able to fix this startup bug?...

**Comment by dshadowwolf** [comment]
...don't know - It's in 1.11.2 and thats @jriwanek's territory as the "queen of backports and long term support" - I'm running things out at the edge of whats available.

**Comment by TheCorinthians** [comment]
Ah, I see... well, take your time, I hope it's not too difficult for you guys... Once again, I'm sorry that's an older version...
